DOCUMENTATION.md

```diff
@@ -104,751 +29,740 @@ import uk.gov.service.notify.SendSmsResponse;
 
-</details>
+```java
+Map<String, String> personalisation = new HashMap<>();
+personalisation.put("first_name", "Amala");
+personalisation.put("application_date", "2018-01-01");
+```
+If a template does not have any placeholder fields for personalised information, you must pass in an empty map or `null`.
 
-#### Arguments
+#### reference (optional)
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+A unique identifier. This reference can identify a single unique notification or a batch of multiple notifications. If you do not have a reference, you must pass in an empty string or `null`.
 
-##### `mobileNumber`
-The mobile number the SMS notification is sent to.
+```
+String reference='STRING';
+```
 
-##### `templateId`
+#### smsSenderId (optional)
 
-The template id is visible on the template page in the application.
+A unique identifier of the sender of the text message notification. To set this up:
 
-##### `reference`
-An optional unique identifier for the notification or an identifier for a batch of notifications. `reference` can be an empty string or null.
+1. Sign into your GOV.UK Notify account.
+1. Go to __Settings__.
+1. If you are not in the correct service, select __Switch service__ in the top right corner of the screen and select the correct one.
+1. Go to the __Text Messages__ section and select __Manage__ on the "Text Message sender" row.
+1. You can do one of the following:
+  - copy the ID of the sender you want to use and paste it into the method
+  - select __Change__ to change the default sender that the service will use, and select __Save__
 
-##### `personalisation`
+```
+String smsSenderId='8e222534-7f05-4972-86e3-17c5d9f894e2'
+```
+
+If you do not have have an `smsSenderId`, you must pass in an empty string or `null`.
+
+### Response
+
+If the request to the client is successful, you will receive the following `SendSmsResponse`:
 
-If a template has placeholders, you need to provide their values, for example:
 
 ```java
-Map<String, String> personalisation = new HashMap<>();
-personalisation.put("name", "Jo");
-personalisation.put("reference_number", "13566");
+UUID notificationId;
+Optional<String> reference;
+UUID templateId;
+int templateVersion;
+String templateUri;
+String body;
+Optional<String> fromNumber;
 ```
 
-#### `smsSenderId`
-Optional. Specifies the identifier of the text message sender to set for the notification. The identifiers are found in your service Settings, when you 'Manage' your 'Text message senders'.
-If you omit this argument your default text message sender will be for the notification.
+If you are using the [test API key](/#test), all your messages will come back as delivered.
+
+All successfully delivered messages will appear on your dashboard.
 
+### Error codes
 
-</details>
+If the request is not successful, the client will raise an `HTTPError`:
```

Comment:
HTTPError is a NotificationClientException

Reply:
Changed